libindy/build.rs

```diff
@@ -32,4 +32,40 @@ fn main() {
             }
         }
     }
+		match &target {
+		x if x.contains("aarch64-linux-android") || x.contains("armv7-linux-androideabi") ||
```

Comment:
Can we just replace this complex if with just "linux-android" as it seems present in all arch strings?